src/azul/service/manifest_service.py

```diff
@@ -841,6 +841,10 @@ def _samples_tsv(self, bundle_tsv: IO[str]) -> None:
             qualifier: Qualifier = file['file_format']
             if qualifier in ('fastq.gz', 'fastq'):
                 qualifier = f"fastq_{file['read_index']}"
+            # Terra requires column headers only contain alphanumeric
+            # characters, underscores, and dashes.
+            # See https://github.com/DataBiosphere/azul/issues/2182
+            qualifier = re.sub(r'[^A-Za-z0-9_-]', '_', qualifier)
```

Comment:
If dash is allowed, we should replace with dash so as not to make parsing the read_index more complicated.